Automattic/wp-calypso
# Help Center: enable HC entry point in staging in editor but not checkout

#### Changes proposed in this Pull Request

* This PR creates a new flag for the help center in checkout and decouples it from the one for the editor. 
* This is needed to start the CfT, since we need the help-center in the master bar in the editor but not in the checkout.
* Then it enables the original flag in staging (so a12ns can see HC in the editor). 

#### Testing instructions

1. Locally, go to http://calypso.localhost:3000/post/YOUR_SITE
2. Simulate mobile in DevTools. You should see the HC entry point. 
3. Close DevTools.
4. Go To http://calypso.localhost:3000/checkout/YOUR_SITE.
5. Re-open DevTools.
6. You shouldn't see the HC entry point.
